sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -1425,6 +1905,22 @@ internal virtual async Task<Response> CheckPermissionAsync(string permission, st
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="permission"/> or <paramref name="connectionId"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="permission"/> or <paramref name="connectionId"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
+        /// This sample shows how to call CheckPermission with required parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new WebPubSubServiceClient(endpoint, credential);
+        /// Response response = client.CheckPermission("permission", "connectionId");
+        /// </code>
+        /// This sample shows how to call CheckPermission with all parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new WebPubSubServiceClient(endpoint, credential);
+        /// Response response = client.CheckPermission("permission", "connectionId", null);
```

Comment:
Curious what the expectation is for the customer to do with this response to check the permission?